src/lib/definitions.js

```diff
@@ -768,6 +768,9 @@ var createParseIntArgDefault;
         ],
         'D':
         [
+            // The escaped character may be either ] or }
+            define('escape((+("10"+(RP_5_N+FILTER+0)[40]+0+0+0)+FILTER)[40])[2]'),
+            define('escape((NaN+(+("10"+(FILL+0)[33]+0+0+0)+FILL))[41])[2]', FILL),
```

Comment:
Use `RP_3_NO` rather than `NaN` to when it's beeing used as a 3-character padding:

```js
'escape((RP_3_NO + (+("10" + (FILL + 0)[33] + 0 + 0 + 0) + FILL))[41])[2]'
```

Also, move this line after `define('(RP_3_NO + document)[11]', DOCUMENT),`. That's because character definitions are sorted by the number of their features, and lexicographically according to the letters of the feature.